cosmic_ray/reporting.py

```diff
@@ -8,7 +8,7 @@
 def _print_item(work_item, full_report):
     data = work_item.data
     outcome = work_item.worker_outcome
-    if outcome in [WorkerOutcome.NORMAL, WorkerOutcome.EXCEPTION]:
+    if outcome in [WorkerOutcome.NORMAL, WorkerOutcome.ABNORMAL, WorkerOutcome.EXCEPTION]:
```

Comment:
Minor: If you use a set here rather than a list, the Python compiler optimises the code.